src/OpenTelemetry.AutoInstrumentation/OpenTelemetry.AutoInstrumentation.csproj

```diff
@@ -18,6 +18,9 @@
     <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.3.0" />
     <PackageReference Include="OpenTelemetry.Shims.OpenTracing" Version="1.0.0-rc9.4" />
     <PackageReference Include="OpenTelemetry.Exporter.Prometheus" Version="1.3.0-rc.2" />
+    <PackageReference Include="OpenTelemetry.Instrumentation.MySqlData" Version="1.0.0-beta.3" Condition="'$(TargetFramework)' == 'netcoreapp3.1'" />
+    <!-- MySql.Data is required by OpenTelemetry.Instrumentation.MySqlData. ExcludeAssets="all" prevents copying it to the output -->
+    <PackageReference Include="MySql.Data" Version="6.10.7" ExcludeAssets="all" Condition="'$(TargetFramework)' == 'netcoreapp3.1'" />
```

Comment:
This approach may not be scalable if we ever need to support multiple versions of an instrumentation library (if those libraries depend on different versions of the library being instrumented).

Reply:
I do not see any _standard_ solution to manage such dependencies. I suppose that we will need to create custom solution to download and load on runtime such dependencies.

Would you like to focus on it right now, or postpone when it will be needed?

Reply:
Let's postpone any exploration until we discover a need. There are ways to solve the problem, and we don't need the added complexity yet.